package.json

```diff
@@ -160,6 +160,17 @@
           ],
           "default": "opa-fmt",
           "description": "The formatter to use for Rego. Supports: ['opa-fmt', 'opa-fmt-rego-v1', 'regal-fix']."
+        },
+        "regal.trace.server": {
```

Comment:
This looks positive, I think it looks too like we might be able to replace https://github.com/open-policy-agent/vscode-opa/blob/bc2404b9ee229457fbb8cb2c69488654e583e6ce/src/ls/clients/regal.ts#L120 with this - but only once we have the Regal LS side to this too?

Do we not need to send/set this config as part of the initialisation options too? (i.e. in here somewhere: https://github.com/open-policy-agent/vscode-opa/blob/main/src/ls/clients/regal.ts#L167)